server/src/project/project.service.ts

```diff
@@ -507,6 +509,40 @@ export class ProjectService {
       console.log(e);
     }
 
+    let projectDispositions =
+      transformedProject[
+        "dcp_dcp_project_dcp_communityboarddisposition_project"
+      ];
+
+    try {
+      projectDispositions = await Promise.all(
+        projectDispositions.map(async disposition => {
+          if (
+            [717170004, 717170003].includes(disposition.dcp_visibility) && // LUP or General Public
```

Comment:
tiny tiny nit: can we use the consts here?

Reply:
yep!